default/python/universe_generation/monsters.py

```diff
@@ -160,11 +160,8 @@ def generate_monsters(monster_freq, systems):
                 tracked_plan_valid_locations[fp] += 1
 
         # filter out all monster fleets whose location condition allows this system and whose counter hasn't reached 0.
-        suitable_fleet_plans = [fp for fp in fleet_plans
-                                if system in fp_location_cache[fp]
-                                and spawn_limits[fp]
-                                and (fp not in fleet_can_alter_starlanes
-                                     or starlane_altering_monsters.can_place_at(system, fp))]
+        suitable_fleet_plans = [fp for fp in fleet_plans if system in fp_location_cache[fp] and spawn_limits[fp] and
+                                (fp not in fleet_can_alter_starlanes or starlane_altering_monsters.can_place_at(system, fp))]
```

Comment:
Just curious: does the original formatting really violate PEP8? Because, honestly, I think the original formatting is much more readable...